### PR TITLE
Added the README.md file for PowerCLI

### PR DESCRIPTION
**Pull request by hkx303**
**Description**
Added the simple README.md file for PowerCLI

### CODE REVIEW

File: hypervisor/virt/esx/README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    server = config.server
+    admin_user = config.admin_user
+    admin_passwd = config.admin_passwd
+    ssh_user = config.ssh_user
+    ssh_passwd = config.ssh_passwd
+    guest_name = config.guest_name
+    esx_host = config.esx_host
+    esx_host_user = config.esx_host_user
+    esx_host_pwd = config.esx_host_pwd
+    image_path = config.image_path
```
Comment (Junefen) [inline hypervisor/virt/esx/README.md]: It's just for virtwho-test project to get necessary values. I think we can just set the null value like ```server=''``` for public settings.

File: hypervisor/virt/esx/README.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+# Hypervisor-builder: ESX hypervisor
```
Comment (Junefen) [inline hypervisor/virt/esx/README.md]: I think here we can also use ```Hypervisor-builder: vCenter```, don't need to match the ```ESX``` mode name in virt-who.

File: hypervisor/virt/esx/README.md (hunk 1 above)
Comment (hkx303) [reply hypervisor/virt/esx/README.md]: Modified.

File: hypervisor/virt/esx/README.md (hunk 2 above)
Comment (hkx303) [reply hypervisor/virt/esx/README.md]: OK

**Review by Junefen — APPROVED** [review]
Looks good.

**Pull request merged**